js/phobos.js

```diff
@@ -23,14 +23,28 @@ function addAnchors()
         // we link to the first children
         var a = items[i].querySelector('a');
         if(!a) continue;
+
+        // copy-to-clipboard
+        var el = document.createElement("i");
+        var symbol = a.name.replace(".", "");
```

Comment:
This was the "safer" choice - it will continue to work if the naming of the anchor link is changed and I do imagine that we want to kill the (unneeded) dot in the future ;-)